datastore/longhorn.go

```diff
@@ -226,12 +226,23 @@ func (s *DataStore) GetSettingValueExisted(sName types.SettingName) (string, err
 	if err != nil {
 		return "", err
 	}
-	if setting.Value == "" {
-		return "", fmt.Errorf("setting %v is empty", sName)
-	}
 	return setting.Value, nil
 }
 
+func (s *DataStore) GetSettingIfExisted(sName types.SettingName) (*longhorn.Setting, error) {
```

Comment:
Why we need another function than `GetSetting`?

Reply:
In the init step, we want to check if the default setting is applied. If not, we cannot deploy engine image, csi attacher, and csi provisioner.

Reply:
It's better not to use `GetSetting`. Because it may initiate the setting with the value hardcoded in the code rather than using the value in the yaml file.

Reply:
... Then `GetSetting` should read from yaml file.